packages/astro/src/assets/fonts/cache.ts

```diff
@@ -1,59 +1,17 @@
-import { existsSync } from 'node:fs';
-import { mkdir, readFile, writeFile } from 'node:fs/promises';
-import { dirname } from 'node:path';
-import { fileURLToPath } from 'node:url';
-import type * as unifont from 'unifont';
-import { AstroError, AstroErrorData } from '../../core/errors/index.js';
-
-type Storage = Required<unifont.UnifontOptions>['storage'];
-
-export const createStorage = ({
-	base,
-}: {
-	base: URL;
-}): Storage => {
-	return {
-		getItem: async (key) => {
-			const dest = new URL('./' + key, base);
-			try {
-				if (!existsSync(dest)) {
-					return;
-				}
-				const content = await readFile(dest, 'utf-8');
-				try {
-					return JSON.parse(content);
-				} catch {
-					// If we can't parse the content, we assume the entry does not exist
-					return;
-				}
-			} catch (e) {
-				throw new AstroError(AstroErrorData.UnknownFilesystemError, { cause: e });
-			}
-		},
-		setItem: async (key, value) => {
-			const dest = new URL('./' + key, base);
-			try {
-				await mkdir(dirname(fileURLToPath(dest)), { recursive: true });
-				return await writeFile(dest, JSON.stringify(value), 'utf-8');
-			} catch (e) {
-				throw new AstroError(AstroErrorData.UnknownFilesystemError, { cause: e });
-			}
-		},
-	};
-};
+import type { Storage } from 'unstorage';
```

Comment:
The PR doesn't say anything about this change. We should either revert it or provide some information in the PR description